README.md

```diff
@@ -109,13 +109,37 @@ Otherwise :violin: kindly ask someone from the dev team to help troubleshoot.
+| Environment variable                              | Default                                     | Description
+|---------------------------------------------------| --------------------------------------------| -----------
+| DEFAULT_WEBSITE_URL                               | "http://localhost:8080"                     | Service URL
+| DEFAULT_SLACK_WARNING_CHANNEL                     | "slack-client-test"                         | Slack channel
+| DEFAULT_SLACK_ALARM_CHANNEL                       | "slack-client-test"                         | Slack alarm channel
+| DEFAULT_SLACK_USERNAME                            | "Zebedee"                                   | Slack user
+| DEFAULT_PUBLIC_WEBSITE_URL                        | "http://localhost:8080"                     | Service public URL
+| DEFAULT_FLORENCE_URL                              | "http://localhost:8081"                     | Florence URL
+| DEFAULT_BRIAN_URL                                 | "http://localhost:8083"                     | Brian URL
+| DEFAULT_TRAIN_URL                                 | "http://localhost:8084"                     | Train URL
+| DEFAULT_DYLAN_URL                                 | "http://localhost:8085"                     | Dylan URL
+| CONTENT_DIRECTORY                                 | "content"                                   | Content directory
+| KAFKA_SEC_PROTO                                   | _unset_                                     | if set to "TLS", kafka connections will use TLS
+| KAFKA_SEC_CLIENT_CERT                             | _unset_                                     | PEM for the client certificate [1]
+| KAFKA_SEC_CLIENT_KEY                              | _unset_                                     | PEM for the client key [1]
+| AUDIT_DB_ENABLED_ENV_VAR                          | "audit_db_enabled"                          | Audit DB
+| MATHJAX_SERVICE_URL                               | "http://localhost:8888"                     | Mathjax service URL
+| DATASET_API_URL                                   | "http://localhost:22000"                    | Dataset API URL
+| IMAGE_API_URL                                     | "http://localhost:24700"                    | Image API URL
+| ENABLE_KAFKA                                      | false                                       | Feature flag to send kafka messages when a collection is published
+| KAFKA_ADDR                                        | "localhost:9092"                            | Comma seperated list of kafka brokers
+| KAFKA_CONTENT_PUBLISHED_TOPIC                     | content-published                           | Kafka topic to send content pubished event to
+| DATASET_API_AUTH_TOKEN                            | "FD0108EA-825D-411C-9B1D-41EF7727F465"      | Dataset API authentication token
+| SERVICE_AUTH_TOKEN                                | "15C0E4EE-777F-4C61-8CDB-2898CEB34657"      | Service API authentication token
```

Comment:
Just checking this is the developer local default and not a prod value?

Reply:
These values are just updated from existing configurations for the purpose of updating Readme.